tools/buildsteps/win32/bootstrap-addons.bat

```diff
@@ -32,6 +35,7 @@ IF "%WORKDIR%" == "" (
 )
 
 rem setup some paths that we need later
+SET BUILD_ON_CORES=8
```

Comment:
There is an environment variable for it in Windows but don't remember the name now